### ISSUE TITLE
flatpak support

### ISSUE THREAD

**Issue by WoozyMasta** [WoozyMasta/dayz-ctl#3]

**Comment by juxuanu** [comment]
Notes for Flatpak:
- instead of `steamcmd`, use `flatpak run --command=steamcmd com.valvesoftware.Steam`
- default location for games is `~/.var/app/com.valvesoftware.Steam/data/Steam/steamapps/common/`

**Comment by WoozyMasta** [comment]
yes, most likely this is so, I don’t have time to deal with flatpak yet, maybe someday I'll do it, or I’ll consider PR
